third_party/move/move-compiler-v2/tests/ability-check/v1-typing/phantom_param_struct_decl_invalid.exp

```diff
@@ -0,0 +1,50 @@
+
+Diagnostics:
+error: phantom type `T` can only be used as an argument for another phantom type parameter
+  ┌─ tests/ability-check/v1-typing/phantom_param_struct_decl_invalid.move:4:12
+  │
+4 │         a: T,
+  │            ^
+  │
+  = required by declaration of field `a`
```

Comment:
This error would be greatly improved if we could show the declaration of `T` as `phantom`.  Please file an issue.